pkg/packages.builds

```diff
@@ -3,7 +3,8 @@
   <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), dir.props))\dir.props" />
 
   <ItemGroup>
-    <Project Include="$(MSBuildThisFileDirectory)\projects\*\*.builds" />
+  	<ProjectExclusions Include="$(MSBuildThisFileDirectory)\projects\*\Microsoft.NETCore.UniversalWindowsPlatform.builds" Condition="'$(OSEnvironment)' != 'Windows_NT'" />
```

Comment:
If we do this at the individual project level it would still require adding conditions on shared code, since if we disable it on the `Microsoft.NETCore.UniversalWindowsPlatform.builds` file it would only disable the identity package build, but not the RID specific ones. They get built based on the [shared dir.props code](https://github.com/dotnet/core-setup/blob/master/pkg/projects/dir.props#L71-L76) where a condition would make less sense to me, which is why I thought that the best approach was to exclude the builds file entirely one level up.